tests/common/include/tests/common/read_geometry.hpp

```diff
@@ -73,9 +75,12 @@ auto read_from_csv(detector_input_files& files,
  *         created container), the module transsforms and the surface masks.
  */
 template <typename surface_t, typename mask_t>
-inline auto create_modules(scalar m_half_x = 8.4, scalar m_half_y = 36.,
-                           scalar m_tilt_phi = 0.145, scalar layer_r = 32.,
-                           scalar radial_stagger = 2., scalar l_overlap = 5.,
+inline auto create_modules(const scalar m_half_x = 8.4,
```

Comment:
Can you move `toy_geometry` related functions to other file? - its not reading

Reply:
Done